ast/misc.go

```diff
@@ -109,7 +109,6 @@ type DeallocateStmt struct {
 	stmtNode
 
 	Name string
-	ID   uint32 // For binary protocol, there is no Name but only ID.
```

Comment:
if we use mysql C API or even go-mysql-sql, we send ComStmtClose with prepare statement ID, how do we drop statement here?